### ISSUE TITLE
SNS Message Publishing Fails (sns_boto.publish Requires Keywords)

### ISSUE THREAD

**Issue by colinbjohnson** [cevoaustralia/aws-backup-lambda#1]
# Note
Will be providing a pull request to address this issue.

# Description of Error
When providing either a `arn` or `error_arn` as an input to the `backuplambda.py` function (see example below):

`event = {"period_label": "day", "period_format": "%a%H-%M", "keep_count": 14, "rds_region_name": "us-west-2", "tag_name": "MakeSnapshot", "tag_value": "True", "arn": "arn:aws:sns:us-west-2:<account_id>:aws-lambda-snapshot", "error_arn": "arn:aws:sns:us-west-2:<account_id>:aws-lambda-snapshot"} 
`
the following error occurs:
```
{
  "stackTrace": [
    [
      "/var/task/backuplambda.py",
      449,
      "lambda_handler",
      "sns_boto.publish(sns_arn, backup_mgr.message, 'Finished AWS RDS snapshotting')"
    ],
    [
      "/var/runtime/botocore/client.py",
      251,
      "_api_call",
      "\"%s() only accepts keyword arguments.\" % py_operation_name)"
    ]
  ],
  "errorType": "TypeError",
  "errorMessage": "publish() only accepts keyword arguments."
}
```
# Resolution
1. Modify the `sns_boto.publish` function calls in `backuplambda.py` to pass values as keyword arguments. 

## Change From (No Keywords)
`sns_boto.publish(sns_arn, backup_mgr.message, 'Finished AWS RDS snapshotting')`

## Change To (With Keywords)
`sns_boto.publish(TopicArn=sns_arn, Message=backup_mgr.message, Subject='Finished AWS RDS snapshotting')`

**Comment by stevemac007** [comment]
Thanks for this issue @colinbjohnson this gives me good feedback to get some actual unit testing of the function into the process.

**Comment by stevemac007** [comment]
Fixed with #2 - thanks for the issue and the drive to get some test coverage underway.
